spec/spec_helper.rb

```diff
@@ -1,14 +1,6 @@
 require 'support/database_cleaner.rb'
 require 'devise'
```

Comment:
According to the [devise docs](https://github.com/plataformatec/devise/wiki/How-To:-Test-controllers-with-Rails-3-and-4-(and-RSpec)#controller-specs) this is the preferred way to include devise in tests.